backend/RestAPI/Controllers/VideosController.cs

```diff
@@ -393,6 +393,8 @@ public async Task<ActionResult> MarkVideosForDeletion([FromBody] List<Guid> ids)
                 return NotFound();
             }
 
+            var markedVideos = new LinkedList<Guid>();
```

Comment:
Another bug. If you go to recycling bin, click on deleted video card, it opens video player page. It should not open this page.